big_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -109,14 +109,17 @@ suite() ->
 %%--------------------------------------------------------------------
 
 init_per_suite(Config) ->
-    case {rpc(europe_node1, mongoose_wpool, get_worker, [redis, global, gd]),
-          rpc(asia_node, mongoose_wpool, get_worker, [redis, global, gd])} of
+    case {rpc(europe_node1, mongoose_wpool, get_worker, [redis, global, global_distrib]),
+          rpc(asia_node, mongoose_wpool, get_worker, [redis, global, global_distrib])} of
         {{ok, _}, {ok, _}} ->
             ok = rpc(europe_node2, mongoose_cluster, join, [ct:get_config(europe_node1)]),
 
+            % We have to pass [no_opts] because [] is treated as string and converted
+            % automatically to <<>>
             escalus:init_per_suite([{add_advertised_endpoints, []},
-                                    {extra_config, []}, {redis_extra_config, []} | Config]);
-        _ ->
+                                    {extra_config, []}, {redis_extra_config, [no_opts]} | Config]);
+        Result ->
+            ct:pal("Redis check result: ~p", [Result]),
```

Comment:
Is it a leftover?

Reply:
I was planning to leave it, just in case. However, if it's annoying, I may remove it. :)

Reply:
I noticed only now that it's printed only on failure. That's fine.